lib/gitlab_api/fetch_projects.rb

```diff
@@ -0,0 +1,20 @@
+module GitlabApi
+  class FetchProjects < GitlabApi::Base
+    SUCCESS_RESPONSE_CODE = 200
```

Comment:
This is a GitLab particularity? Or what is it used for?

Reply:
Since the gitlab api is inconsistent within itself, it seemed the best way to be sure that the response was what we actually expected is this. An call was made to the API, when it works as expected this is the response, so I've set this up in order to compare later with the response.status